usr/local/share/bastille/templates/default/thin/Bastillefile

```diff
@@ -0,0 +1,12 @@
+ARG HOST_RESOLV_CONF=/etc/resolv.conf
+ARG IP
```

Comment:
This is not needed right not, right?

Reply:
We do use `HOST_RESOLV_CONF` (though I had a typo in the script, so it wasn't getting passed).

By specifying them, I was trying to address 2 situations:

1. Make it clear what args will be passed to the template in case the user creates their own templates.
2. Consider the args we may need in the future so they're "known" even before we start to use them.

For example, I just added `JAIL_NAME` because we have it available to us and I can see it being needed by a user-defined template.

A more robust approach would be to create config vars for the template args, but that seems like over-engineering. Also, since `create.sh` applies the template, it wouldn't know where to get the values from.